blockchain/fullblocktests/generate.go

```diff
@@ -1645,23 +1645,24 @@ func (g *testGenerator) nextBlock(blockName string, spend *spendableOut, ticketS
 	// Create the unsolved block.
 	block := wire.MsgBlock{
 		Header: wire.BlockHeader{
-			Version:     1,
-			PrevBlock:   g.tip.BlockSha(),
-			MerkleRoot:  calcMerkleRoot(regularTxns),
-			StakeRoot:   calcMerkleRoot(stakeTxns),
-			VoteBits:    1,
-			FinalState:  finalState,
-			Voters:      numVotes,
-			FreshStake:  numTicketPurchases,
-			Revocations: numTicketRevocations,
-			PoolSize:    uint32(len(liveTickets)),
-			Bits:        g.calcNextRequiredDifficulty(),
-			SBits:       int64(ticketPrice),
-			Height:      nextHeight,
-			Size:        0, // Filled in below.
-			Timestamp:   ts,
-			Nonce:       0, // To be solved.
-			ExtraData:   [36]byte{},
+			Version:      1,
+			PrevBlock:    g.tip.BlockSha(),
+			MerkleRoot:   calcMerkleRoot(regularTxns),
+			StakeRoot:    calcMerkleRoot(stakeTxns),
+			VoteBits:     1,
+			FinalState:   finalState,
+			Voters:       numVotes,
+			FreshStake:   numTicketPurchases,
+			Revocations:  numTicketRevocations,
+			PoolSize:     uint32(len(liveTickets)),
+			Bits:         g.calcNextRequiredDifficulty(),
+			SBits:        int64(ticketPrice),
+			Height:       nextHeight,
+			Size:         0, // Filled in below.
+			Timestamp:    ts,
+			Nonce:        0, // To be solved.
+			ExtraData:    [32]byte{},
+			StakeVersion: uint32(2),
```

Comment:
This must be 0 (and the uint32 cast is not needed since it's just a constant number).  The soft fork must not become active (aka stake version is enforced) until a specific hash/height as determined by the `isMajorityVersion` code.  In any case, that will not happen in these consensus tests unless new tests are specifically added for it, in which case those tests will need to properly set this new stake version.